coalib/output/printers/LogPrinterMixin.py

```diff
@@ -0,0 +1,45 @@
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
Uh I'm not sure if we can really remove LOG_LEVEL yet, since it might by used by other printers ...